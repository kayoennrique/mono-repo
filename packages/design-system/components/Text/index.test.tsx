import { customRender } from "@alura/test-commons/react-testing-libary";
import { Text } from "./index";

const render = customRender();

describe("<Text />", () => {
  it("renders h1", () => {
    const { container } = render(<Text tag="h1">Sample text</Text>);

    expect(container).toMatchSnapshot();
  });
});
