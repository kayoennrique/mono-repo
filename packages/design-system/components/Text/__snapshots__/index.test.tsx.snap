// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> renders h1 1`] = `
<div>
  <h1
    class="Text__StyledText-sc-1ul0hnu-0 gDEZaJ"
  >
    Sample text
  </h1>
</div>
`;
